### ISSUE TITLE
Cleaning up the code?

### ISSUE THREAD

**Issue by ASmoliak** [abaldwin88/roamer#20]
After making a quick glance at the code I think it can be improved by having it follow the PEP 8 guidelines, really simple things such as docstrings for every module/class/method/function, double blank lines before and after class/function and single blank line for methods,

**Comment by abaldwin88** [comment]
Yep for sure.  Please free to open a PR adjusting the pylintrc file and updating the rest of the codebase to bring it in line.  I'm good for pretty much all of the suggestions listed in PEP 8.  I do prefer the max character length per line at 100 though.

Docstrings are on me though.  ( Unless someone else get's really familiar with the code )  I'll write them up when I get a chance but I've got some higher priority things to patch up.

**Comment by ASmoliak** [comment]
This is the first time I'm actually trying to contribute on another project on GitHub, I'm using PyCharm, are there some things I should be aware of first?

**Comment by abaldwin88** [comment]
First you'll want to fork this repo and run a local version for development.  You should be able to follow the steps here:  https://github.com/abaldwin88/roamer/blob/master/CONTRIBUTING.md

I'm not sure if you're familiar with command line linting tools but basically the `pylintrc` file defines syntax checks.  That way anyone who pulls down the repo and uses pylint has the same set of style checks as everyone else regardless of editor.  I also have travis ci set to fail the build if pylint raises anything.

My preference is that when making style changes we update the `pylintrc` file so that pylint throws a warning for the previously written code.

It's been awhile since I've used pycharm but should be able to get pylint working inside it:
 https://stackoverflow.com/questions/38134086/how-to-run-pylint-with-pycharm

Don't hesitate to ask questions or submit a half finished PR for feedback.

**Comment by ASmoliak** [comment]
Thank you, I will check it out, when the changes are made I simply make a pull request, correct? _(after running the unit tests and making sure there's no errors thrown)_

**Comment by abaldwin88** [comment]
Yep correct!  Here's a guide: https://help.github.com/articles/creating-a-pull-request-from-a-fork/
